app/models/discourse_post_event/event.rb

```diff
@@ -57,6 +57,25 @@ def set_next_date
       notify_missing_invitees!
     end
 
+    def set_topic_bump
+      date = nil
+      if reminders.present?
+        reminders.split(',').each do |reminder|
+          type, value, unit = reminder.split('.')
+          next if type != 'bumpTopic' || !validate_reminder_unit(unit)
+
+          date = starts_at - value.to_i.public_send(unit)
+          break
+        end
+      end
+
+      Jobs.enqueue(:discourse_post_event_bump_topic, topic_id: self.post.topic_id, date: date)
```

Comment:
If an event doesn't have a reminder configured, will this job be invoked with `date = nil`?

Reply:
Yes, is not super ideal. Maybe we can remove the job altogether in the future